src/v2/view-builder/ViewFactory.js

```diff
@@ -50,6 +50,12 @@ import EnrollAuthenticatorEmailView from './views/email/EnrollAuthenticatorEmail
 import RequiredFactorEmailView from './views/email/RequiredFactorEmailView';
 import ChallengeAuthenticatorEmailView from './views/email/ChallengeAuthenticatorEmailView';
 
+// app(okta verify)
+import EnrollAuthenticatorOVPollView from './views/ov/EnrollAuthenticatorOVPollView';
+import EnrollOVSwitchChannelView from './views/ov/EnrollOVSwitchChannelView';
```

Comment:
EnrollOktaVerifySelectChannel or EnrollOktaVerifySelectAnotherChannel
just be more consistent with the ion form name